src/routes/docs/Sidebar.svelte

```diff
@@ -111,6 +111,11 @@
                     label: 'Assistant',
                     href: '/docs/tooling/assistant',
                     icon: 'icon-sparkles'
+                },
+                {
+                    label: 'Keyboard',
```

Comment:
```suggestion
                    label: 'The Appwriter',
```

Reply:
@eldadfux shouldn't we address the product by the product name itself ?

Reply:
Yes, we had everything combined so it was Keyboard, but now it makes sense to have the Appwriter page's title be "The Appwriter".